changelogs/fragments/2174-ipa-user-userauthtype-multiselect.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ipa_user - Fix ``userauthtype`` option to take in list of strings for the multi-select field instead of single string (https://github.com/ansible-collections/community.general/pull/2174).
```

Comment:
```suggestion
  - ipa_user - fix ``userauthtype`` option to take in list of strings for the multi-select field instead of single string (https://github.com/ansible-collections/community.general/pull/2174).
```